### DISCUSSION TITLE
Github Actions GITHUB_EVENT_PATH does not contain relevant PR information

### DISCUSSION THREAD

**Discussion by xdrdak** [community/community#26130] [Actions]
<p>Hey there,</p>
<p>Been having a super great time with github actions until I’ve hit this weird issue where the json payload contained within <code>$GITHUB_EVENT_PATH</code> environment variable didn’t contain any relevant information about the associated PR.</p>
<p>This leads to some funky things, like being unable to assert PR labels or the PR number when using the provided “actions/bin/filter”.</p>
<p>Is there a way to solve this asides from manually trying to fetch the PR data via the API?</p>

**Comment by gimenete** [comment, marked as answer]
<p>Hi,</p>
<p>The file pointed by the <code>$GITHUB_EVENT_PATH</code> env variable contains the same information than the corresponding webhook event. You can check the reference here <a href="https://developer.github.com/webhooks/#events" rel="nofollow noopener">https://developer.github.com/webhooks/#events</a></p>
<p>Any additional information needs to be programmatically fetched using the API. You can authenticate against the API using the <code>$GITHUB_TOKEN</code>.</p>
<p>Adding PR information to some endpoints and webhook events is in our radar, but we don’t have an estimated release date for these changes, sorry.</p>
